packages/frontend/src/components/LightdashVisualization/VisualizationProvider.tsx

```diff
@@ -34,13 +36,13 @@ const Context = createContext<VisualizationContext | undefined>(undefined);
 
 type Props = {
     chartType: ChartType;
-    chartConfigs: ChartConfig['config'];
+    chartConfigs: CartesianChart;
```

Comment:
I think this can be either `CartesianChart | BigNumberChart | undefined` now you've added the new type

Reply:
Unless we will only use the VizProvider for cartesian charts?